sherpa/ui/utils.py

```diff
@@ -14049,8 +14168,37 @@ def plot(self, *args, **kwargs) -> None:
 
         >>> plot("bkg", 1, "up", "bkg", 1, "down")
 
+        Draw both data sets in black, but with partial opacity:
```

Comment:
This actually draws only one dataset, but with two representations (data and model) - they both have the same id (the one for `get_default_id`)

Reply:
Nope, it draws two datasets - the "up" and the "down" background components for dataset 1.

Reply:
Your comment actually says "both are for 'dataset 1`". So it's about what is a "dataset"? How about 
```suggestion
        Draw data and model for dataset 1 in black, but with partial opacity:
```

Reply:
It doesn't draw the model at all. It plots the background dataset components "up" and "down" for dataset 1. I'll address try to improve this in a commit rather than try to address this via the GH web page...

Reply:
Argh - I'm arguing about the code from the previous example....

Reply:
I've basically taken your suggestion here now, and also added a few other changes to hopefully better-describe functionality (both as examples and as general text in the notes section).